src/pages/namespace/Settings/Broadcasts/index.tsx

```diff
@@ -0,0 +1,227 @@
+import {
+  Table,
+  TableCell,
+  TableHead,
+  TableHeaderCell,
+  TableRow,
+} from "~/design/Table";
+
+import { BroadcastsSchemaType } from "~/api/broadcasts/schema";
+import { Card } from "~/design/Card";
+import { Checkbox } from "~/design/Checkbox";
+import { FC } from "react";
+import { Radio } from "lucide-react";
+import { TableBody } from "@mui/material";
+import { useBroadcasts } from "~/api/broadcasts/query/useBroadcasts";
+import { useTranslation } from "react-i18next";
+import { useUpdateBroadcasts } from "~/api/broadcasts/mutate/updateBroadcasts";
+
+const leftColClasses = "col-span-2";
+const labelCellClasses = "place-self-center px-2";
+const switchCellClasses = "place-self-center px-2";
+const rowClasses = "grid grid-cols-5 gap-0 lg:pr-8 xl:pr-12";
+
+const Broadcasts: FC = () => {
+  const { t } = useTranslation();
+  const { data } = useBroadcasts();
+  const { mutate } = useUpdateBroadcasts();
+
+  if (!data?.broadcast) return null;
+
+  const broadcasts = data.broadcast;
+
+  const toggleBroadcast = (key: keyof BroadcastsSchemaType) =>
+    mutate({
+      payload: {
+        broadcast: {
+          [key]: !broadcasts[key],
+        },
+      },
+    });
+
+  return (
+    <>
+      <div className="mb-3 flex flex-row justify-between">
```

Comment:
The headline is a bit too close to the Card compared to the other components on the page. I guess the reason for that is the missing button on the top right.

Reply:
That's correct. I added padding to the headlines so they have exactly the same height as the button in c973c17ffb35429b9e879a32483c3191c0c86763